docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Great Expectations overview
+---
+
+import CheckpointFlowchart from './images_overview/checkpoint_flowchart.png'
+
+This guide is intended for new users of Great Expectations (GX) and those looking for a more foundational understanding of GX, its components, and its principal workflows.  You will be introduced to the moving parts of GX and how they interact without digging through the code that governs those processes and interactions.  The insight provided in this guide will help you understand the how and why of processes you work through in future tutorials.
+
+## Great Expectations (GX)
+
+GX is a Python library that provides a framework for describing the acceptable state of data and then validating that the data meets those criteria. 
+
+GX enables you to build trust in your data's quality by testing how it performs against your expectations.  Reusable components and ongoing testing provide visibility into the health of your data pipeline over time. The human-readable reports generated by GX allow you to share the results of that analysis with others and better collaborate on improvements to both your data and your understanding of your data.
+
+With GX providing insight into your data and helping assure its quality, you can ensure that your data is fully suitable to be used as intended in your processes.
+
```

Comment:
what's the purpose of the two paragraphs above? is it meant lay out the structure for the rest of the doc? if so, might be better if they're in bullet points. overall feels a bit unnecessary to me though.

Reply:
I was told a couple of times that we needed some kind of value-of-use statement at the start of the doc.  I'll remove this.